pvlib/pvsystem.py

```diff
@@ -1372,6 +1398,67 @@ def get_iam(self, aoi, iam_model='physical'):
         else:
             raise ValueError(model + ' is not a valid IAM model')
 
+    def dc_ohms_from_percent(self):
+        """
+        Calculates the equivalent resistance of the wires using
+        :py:func:`pvlib.pvsystem.dc_ohms_from_percent`
+
+        Makes use of array module parameters according to the
+        following DC models:
+        CEC:
+            `self.module_parameters["V_mp_ref"]`,
+            `self.module_parameters["I_mp_ref"]`,
+        SAPM:
+            `self.module_parameters["Vmpo"]`,
+            `self.module_parameters["Impo"]`,
+        PVsyst-like or other:
+            `self.module_parameters["Vmpp"]`,
+            `self.module_parameters["Impp"]`,
+
+        Other array parameters that are used are:
+        `self.losses_parameters["dc_ohmic_percent"]`,
+        `self.modules_per_string`, and
+        `self.strings`.
+
+        See :py:func:`pvlib.pvsystem.dc_ohms_from_percent` for more details.
+        """
+
+        kwargs = _build_kwargs(['dc_ohmic_percent'],
+                               self.array_losses_parameters)
+
+        # add relevent Vmp and Imp parameters from CEC parameters
+        if all([elem in self.module_parameters
+                for elem in ['V_mp_ref', 'I_mp_ref']]):
+            kwargs.update({'vmp_ref': self.module_parameters['V_mp_ref'],
+                           'imp_ref': self.module_parameters['I_mp_ref']})
```

Comment:
here I would prefer to see explicit assignment (`vmp_ref = `) instead of bundling in kwargs because these are required parameters.